book/book.toml

```
@@ -0,0 +1,20 @@
+[book]
+authors = ["Afonso Lage","Mev Lyshkin"]
+language = "en"
+multilingual = false
+src = "src"
+title = "Bevy ECSS"
+
+[output.html]
+git-repository-url = "https://github.com/afonsolage/bevy_ecss"
+git-repository-icon = "fa-code-fork"
+edit-url-template = "https://github.com/afonsolage/bevy_ecss/edit/master/book/{path}"
```

```suggestion
edit-url-template = "https://github.com/afonsolage/bevy_ecss/edit/main/book/{path}"
```
We use `main` as default branch